packaging/server/build-backend.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
I'd rather move this script in a subfolder (`bin` ? `res` ? `resources` ?)
The idea is to make it obvious this is a resource for the dockerfile and not a script that should be directly run by the dev

Reply:
done